### ISSUE TITLE
Landing page outside pelican

### ISSUE THREAD

**Issue by bravegnu** [Chennaipy/website#47]
The current front page of the site, has a layout that is limiting. An example is, how do you add a lightning talks suggestions page? The sidebar has accumulated too many links already.

I suggest that we create a landing page outside pelican's control, which could be based on HTML and pure.css. Something on the lines of http://robotframework.org/

We will be having separate sections for all members (things mailing list guidelines), for volunteers (talk topics, volunteering guidelines, FAQs), etc.

**Comment by shrayasr** [comment]
> The current front page of the site, has a layout that is limiting. An example is, how do you add a lightning talks suggestions page? The sidebar has accumulated too many links already.

You're right. This was the same problem I was talking about when adding the _General Announcements_ page. It is really limiting. 

> I suggest that we create a landing page outside pelican's control, which could be based on HTML and pure.css. Something on the lines of http://robotframework.org/

Sounds good to me. You mean we can take some input from robotframework for designing our landing page? 

> We will be having separate sections for all members (things mailing list guidelines), for volunteers (talk topics, volunteering guidelines, FAQs), etc.

:+1:

**Comment by bravegnu** [comment]
> Sounds good to me. You mean we can take some input from robotframework for
> designing our landing page? 

Yes. The site's design. The same can be found at PyCon 2014 website http://in.pycon.org/2014/

**Comment by shrayasr** [comment]
> Yes. The site's design. The same can be found at PyCon 2014 website http://in.pycon.org/2014/

Oh I see. Didn't know that. Their index page isn't populated by jekyll you mean? 

In any case, we can go ahead and work on this :)

**Comment by bravegnu** [comment]
Some preliminary work has been done, and is available at #67.

**Comment by shrayasr** [comment]
@Chennaipy/web-team we should really pick up on this. there is a LOT of info missing from our home page
